src/app/components/utilities/richtext.scss

```diff
@@ -5,14 +5,10 @@
 [class*='u-richtext'] {
   > * {
     margin-top: 0;
-    margin-bottom: $unit * 1.5;
-
-    @media #{$mq-md} {
-      margin-bottom: $unit * 2;
-    }
+    margin-bottom: $unit * 1.25;
```

Comment:
Should these changes be wrapped in an `.is-future`?

Reply:
🤦